### DISCUSSION TITLE
Problem in the transferring label from scRNA to scATAC

### DISCUSSION THREAD

**Discussion by ahmadsam66** [stuart-lab/signac#599] [Q&A] [answered]
Hi 

I wanted to transfer scRNA labels to scATAC using the code here : https://satijalab.org/signac/articles/mouse_brain_vignette.html

I gave labels to my scRNA cluster using following :
```
new.cluster.ids <- c("Class1","Class2","Class3","Class4","class5")
names(new.cluster.ids) <- levels(pbmc)
pbmc <- RenameIdents(pbmc, new.cluster.ids)
DimPlot(pbmc, reduction = "umap", label = TRUE, pt.size = 0.5) + NoLegend()

```

Then I tried following for transferring labels :

```
predicted.labels <- TransferData(
  anchorset = transfer.anchors,
  refdata = pbmc$subclass,
  weight.reduction = brain[['lsi']],
  dims = 2:30
)
```

It shows I have no subclass in meta data and error appear. 

How can I fix this ? or introduce subclass for scRNA object?

Thanks

**Comment by timoast** [comment, marked as answer]
Answered here: https://github.com/satijalab/seurat/issues/4416
